main.go

```diff
@@ -123,6 +123,9 @@ var Config struct {
 
 	// MaxConcurrentReconciles is the maximum number of concurrent Reconciles which can be run
 	MaxConcurrentReconciles uint `envconfig:"MAX_CONCURRENT_RECONCILES" default:"3"`
+
+	// AutoApprovalprocess enable auto approval on devices
+	AutoApproval bool `envconfig:"AUTO_APPROVAL_PROCESS" default:"true"`
```

Comment:
I've got doubts here about `true` being the default. IMO the default should be what user is recommended to use and auto approval should be an exceptional case.

Reply:
Mainly because of the use cases, when someone uses for the first time, it's easy to get the workload running without much effort. It's more to reduce the friction, something which I think that it's needed.